_includes/pbc/all/install-features/latest/install-the-ssp-asset-management-feature.md

```diff
@@ -36,11 +29,12 @@ Make sure the following packages are now listed in `composer.lock`:
 
 Add the following configuration to `config/Shared/config_default.php`:
 
-| CONFIGURATION                             | SPECIFICATION                                       | NAMESPACE                               |
-|-------------------------------------------|-----------------------------------------------------|-----------------------------------------|
-| FileSystemConstants::FILESYSTEM_SERVICE   | Flysystem configuration for file management.        | Spryker\Shared\FileSystem               |
-| SelfServicePortalConstants::BASE_URL_YVES | Yves URL used in image URLs.                        | SprykerFeature\Shared\SelfServicePortal |
-| SelfServicePortalConfig::getStorageName() | Defines the Storage name for asset Flysystem files. | SprykerFeature\Zed\SelfServicePortal    |
+| CONFIGURATION                                                                | SPECIFICATION                                       | NAMESPACE                               |
+|------------------------------------------------------------------------------|-----------------------------------------------------|-----------------------------------------|
+| FileSystemConstants::FILESYSTEM_SERVICE                                      | Flysystem configuration for file management.        | Spryker\Shared\FileSystem               |
```

Comment:
Do we need to say not only what it is but why it's needed?

Reply:
Usually, we don't write that.